.github/workflows/add-remove-new-fulcio.yaml

```diff
@@ -24,13 +24,15 @@ jobs:
       fail-fast: false # Keep running if one leg fails.
       matrix:
         k8s-version:
-        - v1.25.x
+          - v1.25.x
 
         leg:
-        - fulcio-key-rotation
+          - fulcio-key-rotation
 
         go-version:
-        - 1.18
+          - 1.18
+          - 1.19
+          - '1.20'
```

Comment:
nit: quote all the versions or none.

Reply:
it only require for `1.20` because yaml understands that as `1.2`

but will update all to use the ticks